spring-cloud-commons/src/main/java/org/springframework/cloud/client/discovery/simple/SimpleDiscoveryClientAutoConfiguration.java

```diff
@@ -68,7 +67,7 @@ public SimpleDiscoveryProperties simpleDiscoveryProperties() {
 	}
 
 	@Bean
-	@Order(Ordered.LOWEST_PRECEDENCE)
+	@Order
```

Comment:
Why this change?

Reply:
This is not necessary as `Ordered.LOWEST_PRECEDENCE` is the default value.

Reply:
so it is!